server/.env.example

```diff
@@ -0,0 +1,6 @@
+PORT=3000
+NODE_ENV=development
+
+# Supabase (required based on dependencies)
+SUPABASE_URL=your_supabase_url
```

Comment:
It is an example file for the .env that can be uploaded to the repository. The applications usually provide it, otherwise, there is no clue about how the .env file looks like.

I adjusted the variable names.